client/src/components/Nav/index.jsx

```diff
@@ -7,22 +7,41 @@ import NavMenuIcon from './NavMenuIcon';
 const color={ backgroundImage: 'url("http://bit.ly/2b2ePzs")', width: "100%", opacity: 0.6 };
 
 class AppNavBar extends Component {
+  constructor(props, context) {
+    super(props);
+    this.logIn = this.props.logIn.bind(this);
+    this.logOut = this.props.logOut.bind(this);
+    this.clearSessions = this.clearSessions.bind(this);
+    console.log("this.props.user", this.props.user);
+  }
 
   componentDidMount() {
+    // this get request will keep the user logged with their own instruments...
+    $.get("/userLoggedInToMakeInst", (resp, err) => {
+      // console.log('this the the resp to userloggedintomakeinst', resp);
+      if (resp[0] == null) {
```

Comment:
keeps user logged in on refresh or HTTP request. 
I put it in the nav bar because it renders on page refresh every time. 
On the server side, it takes advantage of sessions.